releasetool/commands/tag/python.py

```diff
@@ -34,6 +34,8 @@ class Context(releasetool.commands.common.GitHubContext):
     release_version: Optional[str] = None
     release_notes: Optional[str] = None
     github_release: Optional[dict] = None
+    kokoro_job_name: Optional[str] = None
```

Comment:
@busunkim96 it probably makes sense for us to have a common base context for all `tag` commands.